src/pages/help-centre/dbot.tsx

```diff
@@ -41,7 +41,7 @@ const ImageWrapper = styled.div`
     padding: 2.4rem 10.2rem;
     max-width: 60rem;
     width: 100%;
-    margin: 0 10rem;
+    margin: 0 auto;
```

Comment:
This will change the desktop margin as well, better to  change margin to auto just for mobile view :pray: